src/platforms/common/usage/distributed-tracing/custom-instrumentation.mdx

```diff
@@ -9,6 +9,13 @@ supported:
   - android
   - ruby
   - dotnet
+  - javascript
+  - node
+notSupported:
+  - javascript.nextjs
+  - javascript.remix
+  - javascript.sveltekit
+  - javascript.astro
```

Comment:
the idea was that if there is no need to ever do the custom instrumentation there because that is setup anyhow OOTB, then no need to show it (as users may be confused and try to set it up manually either way) - BUT I am not sure if this is true, or if it is a realistic scenario for these SDKs to setup distributed tracing manually?